Library/Sender.ts

```diff
@@ -136,6 +157,8 @@ class Sender {
                             }
                             // store to disk in case of burst throttling
                         } else if (
+                            res.statusCode === 401 || // Unauthorized
+                            res.statusCode === 403 || // Forbidden
```

Comment:
When are the cases where 403 is returned?

Reply:
This is just a place holder, not sure which responses will be send by Breeze most likely will drop 403 check